e2e-tests/README.md

```diff
@@ -24,11 +29,28 @@ npm run test
 npx playwright show-report
 ```
 
+- If you have any UI changes and want to update the snapshots, use the following commands:
+
+```
+npm run update-snapshots
+```
+For more information about playwright snapshot testing, see the official [Playwright Documentation](https://playwright.dev/docs/test-snapshots).
+
+## End to End tests in the pipeline
+
+- Every change to a PR will trigger e2e tests workflow
+
+- If the PR has any UI changes, adding a 'ui change' label is necessary to make sure that the tests are run against the updated UI screenshots
+
+- Once the 'ui change' label is added, the update snapshot workflow will be triggerd. If any UI changes are found and if the snapshots has to be updated, the workflow will push a commit to your PR with the updated snapshots.
+
 ## Folder structure
 
 If you are adding new tests for samples, please follow this folder structure:
-.
-├── tests # Tests for the sample folder
-├── playwright.config.ts # playwright config file
-├── package.json  
-└── README.md
+
+    .
+    ├── tests                                  # End to end tests
```

Comment:
Would you check if this directory tree works on forked repo? I tried it on VS code preview, it looks good but it doesn't look okay in actual repo.

![image](https://user-images.githubusercontent.com/94404633/175439215-33f83201-9e90-400a-aaa1-55bff5c1008d.png)

Reply:
now with this change it looks fine